.github/workflows/ci.yml

```diff
@@ -81,3 +81,21 @@ jobs:
         run: cargo install cargo-apk
       - name: Build APK
         run: cargo apk build --example android
+
+  markdownlint:
+    runs-on: ubuntu-latest
+    steps:
+      - uses: actions/checkout@v2
+        with:
+          # Full git history is needed to get a proper list of changed files within `super-linter`
+          fetch-depth: 0
+
+      - name: Run Markdown Lint
+        uses: github/super-linter@v3
+        env:
+          VALIDATE_ALL_CODEBASE: false
+          LINTER_RULES_PATH: /
```

Comment:
I suggest removing the `LINTER_RULES_PATH` line.  The default value (`.github/linters`) has the advantage of both not being in the root of the repository and of being the place folks expect to find the linter configs when they use this common GitHub Action.

Reply:
I did this for two reasons:
* vscode markedownlint disables the `MD013/line-length` Rule, but super-linter doesn't. So if you want to use both the File is necessary.
* The extension needs to have the File in the Root to have Setting correctly applied
  * `The rules defined by .markdownlint{.jsonc,.json,.yaml,.yml,rc} apply to Markdown files in the same directory and any sub-directories without their own .markdownlint{.jsonc,.json,.yaml,.yml,rc}.`

If the only thing we do is disable the Rule `MD013/line-length` for super-linter, then this can be still be done.

Reply:
In general, I don't believe the needs of any particular user's IDE (or its plugins) should be an overriding factor in CI decisions unless the Bevy project decides to optimize for that particular IDE environment.  I say this even as a VS Code user myself, though not exclusively.

However, I take your point and I can see you've put a lot of thought into this.  If I understand correctly, you are saying that if you take my suggested approach, the contents of the config file would only apply to markdown files inside `.github/linters`?  That would certainly make users of the plugin see different behavior than the linter, which would be annoying.

One intriguing idea is that we might be able to get the best of both worlds by requesting that the [`markdownlint` plugin](https://github.com/DavidAnson/vscode-markdownlint) add support for treating config files within a `.github/linters/` subdirectory the same as a config file located in the current directory.  I think this suggestion has a good chance of being received well given that [GitHub Codespaces](https://github.com/features/codespaces) is powered by VS Code, which may lead to a sizable overlap of developers who use both the `markdownlint` plugin and the `github/super-linter@v3` GitHub Action.

In any case, I accept that my suggestion may not be the best short-term solution here.

Reply:
An understandable Position, which convinced me.

---

Yes, that would be the problem if the config File did more than disabling the Rule `MD013`. 

Looking further into the Documentation I found that it is possible to extend a Config with another in a different location.
In our case this would look like this:
```json
{
  "extends": "./.github/linters/.markdown-lint.yml"
}
```
I also documented this in the new `docs/linters.md` File.